grafana-plugin/src/pages/incident/Incident.tsx

```diff
@@ -777,7 +779,7 @@ function GroupedIncident({
           </div>
         </div>
         <div
-          className={cx('message')}
+          className={cx('message', 'text-secondary')}
```

Comment:
you can use

 <Text type="secondary" className={cx('message')}  dangerouslySetInnerHTML={{
            __html: sanitize(incident.render_for_web.message),
          }}